tests/unittests/test_model_zoo.py

```diff
@@ -411,6 +411,15 @@ def test_quantized_ssd_models():
     _test_model_list(model_list, ctx, x)
 
 
+@unittest.skip("temporarily disabled to fallback to non-mkl version")
+@with_cpu(0)
+def test_quantized_fcn_models():
+    model_list = ['fcn_resnet101_voc_int8', 'fcn_resnet101_coco_int8-symbol']
```

Comment:
maybe `model_list = ['fcn_resnet101_voc_int8', 'fcn_resnet101_coco_int8']`

Reply:
Fixed.

Reply:
> @zhanghang1989 Could you please help take a review at this PR again? Thanks

Could you keep the test function the same as the original to avoid introducing potential issues?